.github/workflows/OCV-PR-4.x-RISCV.yaml

```diff
@@ -0,0 +1,210 @@
+name: OCV PR:4.x RISC-V
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-RISCV.yaml'
+  workflow_call:
+
+concurrency:
+  group: OCV-PR-4.x-RISCV-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DCMAKE_TOOLCHAIN_FILE=$HOME/opencv/platforms/linux/riscv64-clang.toolchain.cmake -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON'
+  PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+  PR_AUTHOR_FORK: ${{ github.event.pull_request.head.repo.full_name }}
+  SOURCE_BRANCH_NAME: ${{ github.head_ref }}
+  TARGET_BRANCH_NAME: ${{ github.base_ref }}
+  ANT_HOME: '/usr/share/ant'
+  GIT_CACHE_DOCKER: '/home/ci/git_cache'
+  PYTHONPATH: /home/ci/build/python_loader:$PYTHONPATH
+  OPENCV_TEST_DATA_PATH: '/home/ci/opencv_extra/testdata'
+  OPENCV_DOCKER_WORKDIR: '/home/ci/opencv'
+  DNN_MODELS: '/home/ci/dnn-models'
+  PARALLEL_JOBS: 16
+  TEST_RUNNER: '/opt/tools/qemu-riscv64 -cpu syntacore-scr9 -L /opt/riscv/sysroot'
```

Comment:
path is incorrect. `/opt/tools/bin/qemu-riscv64`.

Reply:
Fixed, thank you.